packages/backend/migrations/src/migrations/20250331143231-init.ts

```diff
@@ -0,0 +1,274 @@
+import { type Kysely, sql } from 'kysely';
+
+import type { DB } from '@app/shared';
+
+export async function up(db: Kysely<DB>): Promise<void> {
+  // Migration code that update the database to the desired state.
+  await db.transaction().execute(async (trx) => {
+    await sql`
+      ALTER SCHEMA mingo  DEFAULT COLLATE utf8mb4_general_ci ;
+    `.execute(trx);
+
+    await sql`
+      CREATE TABLE role (
+        id INT PRIMARY KEY AUTO_INCREMENT,
+        name VARCHAR(30) NOT NULL UNIQUE
+      );
+      `.execute(trx);
+
+    await sql`
+      CREATE TABLE account_status (
+        id INT PRIMARY KEY AUTO_INCREMENT,
+        name VARCHAR(30) NOT NULL UNIQUE
+      );
+      `.execute(trx);
+
+    await sql`
+      CREATE TABLE user (
+        id INT PRIMARY KEY AUTO_INCREMENT,  
+        username VARCHAR(30) NOT NULL UNIQUE,
+        mail VARCHAR(255) NOT NULL UNIQUE,
+        password VARCHAR(255) NOT NULL,
```

Comment:
```suggestion
        password VARCHAR(72) NOT NULL,
```
Un mot de passe hashé avec bcrypt ne sera jamais plus grand que 72 caractères

Reply:
On en reparle ;)